bridges/snowbridge/pallets/ethereum-client/src/impls.rs

```diff
@@ -76,7 +82,8 @@ impl<T: Config> Pallet<T> {
 		receipts_root: H256,
 		proof: &Proof,
```

Comment:
Rather have a more specific, narrowed inputs:

```suggestion
		receipt_proof: (Vec<Vec<u8>>, Vec<Vec<u8>>),
```

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/125/commits/ecd013df4c54c11bd0efa497166a0e89e209689b